src/encoded/__init__.py

```diff
@@ -129,6 +130,23 @@ def app_version(config):
 
         config.registry.settings['snovault.app_version'] = version
 
+    # GA Config
+    ga_conf_file = config.registry.settings.get('ga_config')
+    if not ga_conf_file:
+        raise Exception("No ga_config (path to JSON file of config) define in config. Aborting.")
```

Comment:
I don't think we can do this as it will break the local deployment? Might require a second setting to indicate that we'd like GA on. You can just add `use_GA` to `production.ini` and set it to False in `base.ini`, although TBH the way we use/organize these init files is a little strange and is probably going to change

Reply:
The file is in repo so I think it should be fine.. has per-hostname config.
But yeah, will prly modify to not raise exception. Just prevent downloads unless analytics exists :-P.